RegistrationLib/__init__.py

```diff
@@ -3,7 +3,11 @@
 from Landmarks import *
 from RegistrationState import *
 from RegistrationPlugin import *
-from AffinePlugin import *
-from ThinPlatePlugin import *
-from LocalBRAINSFitPlugin import *
-from LocalSimpleITKPlugin import *
+
+for plugin in [
+  'Affine',
+  'ThinPlate',
+  'LocalBRAINSFit',
+  'LocalSimpleITK'
+  ]:
+  registerRegistrationPlugin('RegistrationLib.%sPlugin' % plugin)
```

Comment:
What's the syntax for a plugin that comes from an extension?  E.g. plastimatch provides a plugin via slicerrt.